packages/spatial/src/physics/components/ColliderComponent.tsx

```diff
@@ -49,7 +50,8 @@ export const ColliderComponent = defineComponent({
       friction: 0.5,
       restitution: 0.5,
       collisionLayer: CollisionGroups.Default,
-      collisionMask: DefaultCollisionMask
+      collisionMask: DefaultCollisionMask,
+      hasCollider: false
```

Comment:
let's scope hasCollider to the reactor as to not expose it on the component properties

Reply:
implemented